tasks/create_app_builds.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Create BuildConfigs for an app
+
+- name: Print app name
```

Comment:
But in fact we need a different build for each customer and it needs the address and ssh key to the private repo with the theme for this customer?